ehrql/tables/beta/tpp.py

```diff
@@ -231,80 +247,180 @@ class clinical_events(EventFrame):
 
 @table
 class ec(EventFrame):
+    """
+    Emergency care admissions data is provided via the NHS Secondary Uses Service.
+
+    This table gives core details of admissions.
+    """
+
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Primary key used to join to the other EC tables",
     )
     arrival_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient self presented at the accident & emergency department, or arrived in an ambulance at the accident & emergency department.",
     )
     sus_hrg_code = Series(
         str,
-        description="TODO",
+        description="The core HRG derived by sus+, as a result of HRG grouping, and used for tariff application.",
     )
 
 
 @table
 class ec_cost(EventFrame):
+    """
+    Emergency care admissions data is provided via the NHS Secondary Uses Service.
+
+    This table gives details of admission costs.
+    """
+
     ec_ident = Series(
         int,
         constraints=[Constraint.NotNull()],
-        description="TODO",
+        description="Primary key used to join to the other EC tables",
     )
     grand_total_payment_mff = Series(
         float,
-        description="TODO",
+        description="The total payment for the activity (Net_SLA_Payment + Tariff_MFF_Payment).",
     )
     tariff_total_payment = Series(
         float,
-        description="TODO",
+        description="The total payment according to the national tariff.",
     )
     arrival_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient self presented at the accident & emergency department, or arrived in an ambulance at the accident & emergency department.",
     )
     ec_decision_to_admit_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date a decision to admit was made.",
     )
     ec_injury_date = Series(
         datetime.date,
-        description="TODO",
+        description="The date the patient was injured.",
     )
 
 
 @table
 class emergency_care_attendances(EventFrame):
-    id = Series(int)  # noqa: A003
-    arrival_date = Series(datetime.date)
-    discharge_destination = Series(SNOMEDCTCode)
+    """
+    Emergency care admissions data is provided via the NHS Secondary Uses Service.
+
+    This table gives details of admission attendances.
+    """
+
+    id = Series(  # noqa: A003
+        int,
+        constraints=[Constraint.NotNull()],
+        description="Primary key used to join to the other EC tables",
```

Comment:
As before, a user can't join to the other EC tables, can they? Could we describe this series as an identifier or an ID?